test/NuGet.CommandLine.Staging.UnitTests/StagingCommandUnitTests.cs

```diff
@@ -336,7 +353,422 @@ public void ListCommand_WhenStagesExistTheyArePrinted()
             }
         }
 
-        private MockServer CreateV3Server(JObject indexJson)
+        [Fact]
+        public void GetCommand_WhenStagingServiceReturnsAnErrorItIsShown()
+        {
+            // Arrange
+            const string stageId = "1323";
+
+            var indexJson = Util.CreateIndexJson();
+
+            using (var serverV3 = CreateV3Server(indexJson))

```

Comment:
It might be nice if you could rid of a lot of the repeated code in these tests by making a method that you could call with a number of parameters.

Every test appears to follow this structure...
```
var indexJson = Util.CreateIndexJson();
using (var serverV3 = CreateV3Server(indexJson))
{
    using (var stagingServer = new MockServer())
    {
         Util.AddStagingResource(indexJson, stagingServer);
         
         // A: Make a call to the staging server API

         serverV3.Start();
         stagingServer.Start();

         string[] args = /* B: Initialize the args for some command */ ;
         var result = CommandRunner.Run(NuGetExe, Directory.GetCurrentDirectory(), string.Join(" ", args), true);

         // C: Do some asserts
     }
}
```

If you could create a method to pass a series of `Func`s to be run in the place of `A`, `B`, and `C` I think you could shorten a lot of the code significantly.